### PR TITLE
Successful submit alert draft 1

### PR DESCRIPTION
**Pull request by DaveyDevs**
Here is an attempt at the alert message on a successful form submission. I wasn't able to test on the most current repo, so please let me know if this isn't working properly. Thanks!

### CODE REVIEW

File: client/src/components/DirectoryAddUpdate.jsx [hunk 1]
```diff
@@ -13,28 +13,30 @@ import Country from "./Country.jsx";
...
+  const [addressPublic, setAddressPublic] = useState("No");
   const [city, setCity] = useState("");
   const [state, setState] = useState("IL");
   const [zip, setZip] = useState("");
   const [county, setCounty] = useState("");
   const [country, setCountry] = useState("US");
   const [websites, setWebsites] = useState("");
   const [contactName, setContactName] = useState("");
-  const [contactNamePublic, setContactNamePublic] = useState("no");
+  const [contactNamePublic, setContactNamePublic] = useState("No");
```
Comment (laredotornado) [inline client/src/components/DirectoryAddUpdate.jsx:33]: As we use the magic string "No", in several places, let's define it as a constant in the src/utils/constants.jsx file and reference it here.

File: client/src/components/DirectoryAddUpdate.jsx [hunk 2]
```diff
@@ -54,6 +56,9 @@ export default function DirectoryAddUpdate() {
   // While loading coop data from ID
   const [loadingCoopData, setLoadingCoopData] = React.useState(false);
 
+  // Alert provider state
+  const [open, close] = useAlert();
```
Comment (laredotornado) [inline client/src/components/DirectoryAddUpdate.jsx:60]: We don't appear to be using the "close" function anywhere.  Do we need it here?  (Like maybe we could just leave it out and have const { open } = useAlert() or whatever the right syntax is.

File: client/src/components/DirectoryAddUpdate.jsx (hunk 1 above)
Comment (DaveyDevs) [reply client/src/components/DirectoryAddUpdate.jsx]: This is done! Let me know if that name works for the variable.

File: client/src/components/DirectoryAddUpdate.jsx (hunk 2 above)
Comment (DaveyDevs) [reply client/src/components/DirectoryAddUpdate.jsx]: Done!

**Review by laredotornado — APPROVED** [review]

**Pull request merged**